packages/wedges/src/tw-plugin/plugin.ts

```diff
@@ -257,70 +273,80 @@ const corePlugin = (
             "wg-line-spinner": "lineSpinner 1.5s ease-in-out infinite both",
           },
           keyframes: {
+            fadeIn: {
+              "0%": {
+                opacity: "0",
+                transform: animateTransform({ scaleX: ".97", scaleY: ".97" }),
+              },
+              "100%": {
+                opacity: "1",
+                transform: animateTransform({ scaleX: "1", scaleY: "1" }),
+              },
+            },
             fadeInUp: {
               "0%": {
                 opacity: "0",
-                transform: "translateY(5px) scale(.97)",
+                transform: animateTransform({ translateY: "5px", scaleX: ".97", scaleY: ".97" }),
               },
               "100%": {
                 opacity: "1",
-                transform: "translateY(0px) scale(1)",
+                transform: animateTransform({ translateY: "0", scaleX: "1", scaleY: "1" }),
```

Comment:
So currently we are hardcoding the final state of the animations which basically is the initial state (`translateY(0), scaleX(1)`).

I was thinking, what if we just remove them and fallback to default variables? Because currently animations override functions defined on the element classes.

For example:
```JSX
<div className="translate-y-1/2 animate-wg-fade-in-up"></div>
```
The animation overrides the translate-y with `0` due to the final keyframe.

Reply:
@brankoconjic ^